### DISCUSSION TITLE
I can't load a save file. Game crashes

### DISCUSSION THREAD

**Discussion by PutoMod** [julianxhokaxhiu/FFNx#491] [Q&A]
**Describe the bug**
- Game crashes when I try to load a save file.

**To Reproduce**
- Game runs perfectly but when I try to load the game:

1) Game Crashes, shows a window saying "Game Crashed :(" and a recommendation to check this page.
2) When I open SYW a window shows saying "It seems the game have crashed previously (crash.ff7 file, etc) it can be related with the Graphic Engine, try other Engines, D3D11/12 and OpenGl mainly for ATI/AMD".
3) Of course I tried ALL THE ENGINES, only D3D12 worked ONCE, never again...

**Expected behavior**
- That the game LOADS THE F***ING SAVE FILE...

**Screenshots**
- I need to create or upload images in other account, I'll pass I prefer to describe what I have enabled in the mod:

SYW V5 Settings:
- HD Fields
- HD Battle
- HD Worldmap
- HD Magics n FXs
- HD Fonts and Menus
- 30 fps FMV
- HD Minigames
- Modern 3D Models
- Transparent Dialog Box
- Transparent Battle Box
- PSX Musics
- True Honey Bee Inn

FFNx Settings:
- Resolution 1920x1080
- HDR Nits: AUTO
- 3D Engine: AUTO (I tried every Engine as I said before)
- Super Sampling x4
- Anti-Aliasing x4
- Use VSync
- 16/9
- Anisotropic Filtering
- Steam Achievments
- Fields Animation: Recommended
- 30fps Battle (yes I tried to load the game without the 30 n 60 fps and putting none in Fields Animation)
- Controls: no auto run, no analog or any custom.

**GPU (please complete the following information):**

- Nvidia GeForce GTX 1050 Ti 
- Driver Version: 30.0.15.1165

**Additional context**
- CPU Intel Core I7 7700
- A friend completed the game, he has an AMD GPU, had some problems loading, but it solved itself by changing engines
- Sorry for any strong language, but I'm really REALLY frustrated with this...

**Comment by PutoMod** [comment]
Update: Apparently, enableling the Steam Achievments causes the load files to crash.
When I dissable it, the game loads the save files; let's hope the problems ends there.

**Comment by julianxhokaxhiu** [comment]
7th issues are not supported here. Feel free to use the Discord support channels.
